package.nls.json

```diff
@@ -12,7 +12,7 @@
     "AWS.configuration.description.logLevel": "The AWS Toolkit's log level (changes reflected on restart)",
     "AWS.configuration.description.onDefaultRegionMissing": "Action to take when a Profile's default region is hidden in the Explorer. Possible values:\n* `add` - shows region in the explorer\n* `ignore` - does nothing with the region\n* `prompt` - (default) asks the user what they would like to do.",
     "AWS.configuration.description.samcli.debug.attach.retry.maximum": "If the Toolkit is unable to attach a debugger, this is the number of times to retry before giving up.",
-    "AWS.configuration.description.samcli.debug.attach.timeout": "Maximum amount of time in milliseconds to wait for local lambda inkokes to start before attempting to attach the debugger.",
+    "AWS.configuration.description.samcli.debug.attach.timeout": "Maximum amount of time to connect a debugger to a SAM Local debug run (in milliseconds)",
```

Comment:
"Maximum amount of time to connect a debugger" reads to me like "maximum amount of time that the debugger may remain attached". What do you think about something like one of these:

* Maximum amount of time to wait for debugger to connect.
* Maximum amount of time to wait for execution to begin.
* Maximum amount of time to wait for execution to begin before timing out.

Reply:
How's `"Maximum amount of time to wait for debugger to connect to a SAM Local debug run (in milliseconds)"` sound?

Reply:
✔️ to the above